lib/mayaUsd/utils/util.h

```diff
@@ -556,10 +556,14 @@ MString convert(const TfToken& token);
 MAYAUSD_CORE_PUBLIC
 std::string convert(const MString&);
 
-} // namespace UsdMayaUtil
+MAYAUSD_CORE_PUBLIC
+MDagPath getDagPath(const MFnDependencyNode& depNodeFn, const bool reportError = true);
```

Comment:
Thanks for making this a public util function. I'm not sure why we hadn't done that to begin with. :/

It looks like there's a third place where the guts of this function are being used as well:
https://github.com/Autodesk/maya-usd/blob/802de0c946012427e9582054d4b7330f58fe5086/lib/mayaUsd/fileio/writeJobContext.cpp#L479

Can I hassle you to clean up the duplicate code there and call this instead?

Reply:
@mattyjams  Awesome. No hassle at all. Will do it soon.

Reply:
@mattyjams  please see 9c0ade65768d9a9266486b0574da98657a48146e